runtime/caml/osdeps.h

```diff
@@ -139,12 +130,14 @@ CAMLextern int caml_win32_isatty(int fd);
 
 CAMLextern void caml_expand_command_line (int *, wchar_t ***);
 
-CAMLextern clock_t caml_win32_clock(void);
-
 CAMLextern value caml_win32_xdg_defaults(void);
 
 CAMLextern value caml_win32_get_temp_path(void);
 
+#include <time.h>
```

Comment:
I'd be inclined to move this to the top of the `_WIN32` section, for consistency.

Reply:
Done, thanks.